client/state/selectors/is-account-recovery-reset-options-ready.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Return a boolean value indicating whether the account recovery reset options are fetched and ready to be used.
+ *
+ * @param  {Object} state Global state tree
+ * @return {Boolean}      If the reset options are ready.
+ */
+export default ( state ) => {
+	const resetOptions = state.accountRecovery.reset.options;
+	return null == resetOptions.error && 0 < resetOptions.items.length;
```

Comment:
> Why 0 < resetOptions.items.length instead of resetOptions.items.length > 0?
It's simply because I'm used to Yoda condition :D 
Especially when the first condition in this expression is already Yoda as `null == resetOptions.error`, it makes me want to put the constant ahead even more.  Would you mind if I keep it this way? Of course, I'm happy to rephrase it if our guidelines are against it :)

> Would there ever be a case where the API returns the reset options, but there are none available?
No. There will be at least the primary one. The primary sms can be empty, but the primary email can't, since that's the essential information one needs to have an account with us.